docs/components/clients/Logo.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import cn from "classnames";
+import Image from "next/future/image";
+import { TurboUser } from "./users";
+
+const DEFAULT_SIZE = {
+  width: 100,
+  height: 75,
+};
+
+export function Logo({
+  user,
+  theme,
+  isLink,
+}: {
+  user: TurboUser;
+  theme: "dark" | "light";
+  isLink: boolean;
+}) {
+  const styles = {
+    ...DEFAULT_SIZE,
+    ...user.style,
+  };
+  const logo = (
+    <Image
+      src={user.image.replace(
+        "/logos",
+        theme === "light" ? "/logos/white" : "/logos/color"
```

Comment:
does this do dark/light mode with JS?

It's possible to do just use html, fwow `<source media="(prefers-color-scheme: dark)" src=...>`

Reply:
It does, I think you need to use picture instead of img for source?

Reply:
@mehulkar , Can you use that site preferences?

  https://turbo-site-git-tomknickman-turbo-259-fix-turborepoorg-sh-c285c1.vercel.sh/#:~:text=Start%20Building%20%E2%86%92-,System,-Footer

Reply:
Nope, `prefers-color-scheme` reflects system usage, same as [the CSS `@media` query](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme). 

> [#:~:text=Start%20Building%20%E2%86%92-,System,-Footer](https://turbo-site-git-tomknickman-turbo-259-fix-turborepoorg-sh-c285c1.vercel.sh/#:~:text=Start%20Building%20%E2%86%92-,System,-Footer)

btw @gaspar09 , I don't use Chrome, so I can't see what you're pointing to with your text fragment links 😛

Reply:
https://www.google.com/chrome/downloads/

Reply:
Did you mean [developer.apple.com/safari/technology-preview/](https://www.youtube.com/watch?v=dQw4w9WgXcQ)??